tests/camera/CMakeLists.txt

```diff
@@ -8,8 +8,11 @@ target_link_libraries(CameraTest
 add_test(NAME CameraTest COMMAND CameraTest)
 
 set(_env_vars)
-list(APPEND _env_vars "GZ_SIM_SYSTEM_PLUGIN_PATH=$<TARGET_FILE_DIR:gz-sim-yarp-camera-system>")
-list(APPEND _env_vars "LIBGL_ALWAYS_SOFTWARE=1")
+list(APPEND _env_vars
+  "GZ_SIM_SYSTEM_PLUGIN_PATH=$<TARGET_FILE_DIR:gz-sim-yarp-camera-system>"
+  "GZ_SIM_RESOURCE_PATH=${GZ_SIM_RESOURCE_PATH}:${CMAKE_CURRENT_SOURCE_DIR}"
```

Comment:
Minor: this takes the `GZ_SIM_RESOURCE_PATH` value when the cmake configuration is run, not when the test is run. If we can require CMake 3.22, we could use https://cmake.org/cmake/help/latest/prop_test/ENVIRONMENT_MODIFICATION.html to add the env variable. Anyhow, if this works let's leave it like these.

Reply:
Thanks, I didn't know about this CMake property. I think there it is fine to advance the CMake required version (also v3.22 was released in 2021). Do you think there are major differences between the two approaches?

Reply:
I think we can do it in a different PR. For the cmake version, 22.04 apt packages uses 3.22 (see https://repology.org/project/cmake/versions), so if we want we can bump to 3.22 .